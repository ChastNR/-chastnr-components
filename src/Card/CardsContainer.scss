.cds__cnt {
  width: 100%;
  margin: 0 auto;
  display: flex;
  flex-wrap: wrap;
  flex-shrink: 0;
  flex-grow: 1;
  flex-basis: 0;
}
