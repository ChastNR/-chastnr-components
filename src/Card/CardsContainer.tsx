import "./CardsContainer.scss";

const CardsContainer: React.FC = ({ children }) => <div className="cds__cnt">{children}</div>;

export default CardsContainer;
