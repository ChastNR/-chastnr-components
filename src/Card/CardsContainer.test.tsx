import { render } from 'enzyme';
import toJson from 'enzyme-to-json';
import { CardsContainer } from './CardsContainer';

describe('<CardsContainer/>', () => {
  it('should match snapshot', () => {
    const cardsContainer = render(<CardsContainer>Test Render</CardsContainer>);
    expect(toJson(cardsContainer)).toMatchSnapshot();
  });
});
