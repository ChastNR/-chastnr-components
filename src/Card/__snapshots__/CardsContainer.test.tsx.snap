// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardsContainer/> should match snapshot 1`] = `
<div
  class="cds__cnt"
>
  Test Render
</div>
`;
